chromadb/ingest/impl/utils.py

```diff
@@ -46,4 +47,4 @@ def trigger_vector_segments_max_seq_id_migration(
 
     for collection_id in collection_ids_with_unmigrated_segments:
         # Loading the segment triggers the migration on init
-        segment_manager.get_segment(UUID(collection_id), VectorReader)
+        segment_manager.get_segment(UUID(collection_id), LocalHnswSegment)
```

Comment:
This is wrong. The segment could be LocalPersistentHnswSegment.

Reply:
Update: the code is correct by coincidence. I'm going to rework on this PR a bit to re-introduce the `VectorReader` and `MetadataReader`